src/main/resources/migrations/00030_add_live_column_to_transaction_table.sql

```diff
@@ -0,0 +1,9 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:add_live_column_to_transaction_table
+
+ALTER TABLE transaction
+    ADD COLUMN live BOOLEAN;
+
+--changeset uk.gov.pay:index_transaction_live_column
+CREATE INDEX transaction_live_idx ON transaction(live)
```

Comment:
```suggestion
CREATE INDEX CONCURRENTLY transaction_live_idx ON transaction(live)
```

Reply:
Not sure if postgres will use index effectively on low cardinality fields. Or we can create partial index

`CREATE INDEX CONCURRENTLY transaction_live_idx ON transaction(live) WHERE live is true;`

Reply:
Great suggestion! A partial index would be much more useful to Postgres, have updated to do this and not run in transaction.